rpcs3/Emu/FS/vfsDirBase.h

```diff
@@ -62,14 +62,12 @@ class vfsDirBase
 
 	public:
 		iterator(vfsDirBase* parent)
-			: parent(parent)
-			, data(parent->First())
+			: data(parent->First())
 		{
 		}
 
 		iterator(const DirEntryInfo* data)
-			: parent(parent)
```

Comment:
if the code is:

class foo(int a)
{
public:
foo(a)
 : a(a) // useless, but this->a(a) is not useless
{
}
int a;
}